code/game/objects/structures/crates_lockers/crates.dm

```diff
@@ -713,3 +713,14 @@
 /obj/structure/closet/crate/secure/zenghu
 	desc = "A secure sterile crate marked with the logo of Zeng-Hu Pharmaceuticals."
 	closet_appearance = /singleton/closet_appearance/crate/branded/zhenghu/secure
+
+//Brings back the old Biohazard Bin.
+/obj/structure/closet/crate/biohazard
+	name = "biohazard bin"
+	desc = "A large bin for medical and other personnel that deal with bio-grade waste."
+	icon = 'icons/obj/storage.dmi'
+	icon_state = "biohazard"
+	climb_allowed = TRUE
+	points_per_crate = 5
+
+/obj/structure/closet/crate/biohazard/LateInitialize()
```

Comment:
why does htis need to lateinitialize?

Reply:
Due to how crates are now coded, if I don't cancel LateInit on this bin it gets converted into one of the new tiny crates, which is what this whole PR is trying to stop.